run.py

```diff
@@ -247,8 +247,27 @@ def rebuild_collection(collection_name):
     logger.info('Done rebuilding collection %s, sent %s records', (collection_name, sent))
 
 
-if __name__ == '__main__':
+def reindex_failed(app):
+    """from status field in records table we compute what failed"""
+    bibs = []
+    with app.session_scope() as session:
+        for rec in session.query(Records) \
+                          .filter(Records.status.notin_(['success', 'retrying'])) \
```

Comment:
I would say that it would be good we also make sure they have bib and non-bib data to be reindexed, otherwise we may end up indexing things that are in the process of being deleted (although the chances are low given that it has to have also an error status).

Reply:
I added a check for bib_data not none, nonbib_data is not required for solr so I didn't want to add it as well.